### PR TITLE
Fix homepage docs link broken

### PR DESCRIPTION
**Pull request by lunny**
And we have no installation-from-package page on docs.gitea.io, we should add one empty at the moment. I will send another PR to do that.

### CODE REVIEW

**Comment by lunny** [comment]
sent go-gitea/docs#17

**Comment by lunny** [comment]
This will fix #404

File: templates/home.tmpl [hunk 1]
```diff
@@ -57,7 +57,7 @@
 					<i class="octicon octicon-flame"></i> 易安装
 				</h1>
 				<p class="large">
-					您除了可以根据操作系统平台通过 <a target="_blank" rel="noopener" href="https://docs.gitea.io/installation/install_from_binary.html">二进制运行</a>，还可以通过 <a target="_blank" rel="noopener" href="https://github.com/go-gitea/gitea/tree/master/docker">Docker</a> 或 <a target="_blank" rel="noopener" href="https://github.com/geerlingguy/ansible-vagrant-examples/tree/master/gogs">Vagrant</a>，以及 <a target="_blank" rel="noopener" href="https://docs.gitea.io/installation/install_from_packages.html">包管理</a> 安装。
+					您除了可以根据操作系统平台通过 <a target="_blank" rel="noopener" href="https://docs.gitea.io/zh-cn/%E4%BB%8E%E4%BA%8C%E8%BF%9B%E5%88%B6%E5%AE%89%E8%A3%85/">二进制运行</a>，还可以通过 <a target="_blank" rel="noopener" href="https://github.com/go-gitea/gitea/tree/master/docker">Docker</a> 或 <a target="_blank" rel="noopener" href="https://github.com/geerlingguy/ansible-vagrant-examples/tree/master/gogs">Vagrant</a>，以及 <a target="_blank" rel="noopener" href="https://docs.gitea.io/en-us/installation-from-package/">包管理</a> 安装。
```
Comment (tboerger) [inline templates/home.tmpl]: Let's fix the URLs first

**Comment by bkcsoft** [comment]
LGTM

**Comment by tboerger** [comment]
LGTM

**Pull request merged**